tools/config/seed.config.ts

```diff
@@ -147,6 +147,12 @@ export class SeedConfig {
   APP_TITLE = 'Welcome to angular-seed!';
 
   /**
+   * Tracking ID.
+   * @type {string}
+   */
+  Google_Analytics_ID = 'UA-XXXXXXXX-X';
```

Comment:
Let's make it uppercase to be consistent with the other configuration properties.